internal/generator/vector/conf_test.go

```diff
@@ -153,6 +156,10 @@ source = """
   del(.stream)
   
   del(.kubernetes.pod_ips)
+  
+  ts = .timestamp
+  del(.timestamp)
+  ."@timestamp" = ts
```

Comment:
you should be able to shorten this to:  `."@timestamp = del(.timestamp) `, since del() returns the value of the deleted field.

Reply:
@cahartma updated PR

Reply:
will need re-lgtm though